modules/services/compton.nix

```diff
@@ -0,0 +1,29 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  options = {
+    services.compton = {
+      enable = mkEnableOption "Compton, a compositor for X11";
+    };
+  };
+
+  config = mkIf config.services.compton.enable {
+    systemd.user.services.compton = {
+        Unit = {
+          Description = "Compton";
```

Comment:
Can expand the description slightly, maybe "Compton X11 compositor" or something?